### PR TITLE
Implement Missions leaving and joining action

### PR DESCRIPTION
**Pull request by TesheMaximillan**
👋 Hi @Have-Samuel 

# In this milestone I did the following tasks

* Switch badges for missions - conditional components
* Implement mission leaving actions
* Implement mission joining actions
* organize project

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import './App.scss';
 import Navbar from './components/common/Navbar';
-import Myprofile from './components/Myprofile';
```
Comment (Have-Samuel) [inline src/App.js:5]: I think it's better to remove the unwanted code to make it clean and readable

File: src/components/missions/Mission.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  } = mission;
+
+  const handleStatus = () => dispatch(updateMissionStatus(mission_id, !status));
+
+  return (
+    <tr>
+      <td>{mission_name}</td>
+      <td>{description}</td>
+      <Status status={status} handleStatus={handleStatus} />
+    </tr>
```
Comment (Have-Samuel) [inline src/components/missions/Mission.jsx:21]: I like the way you apply your styles in the reactjs files. :+1:

File: src/components/missions/Status.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  if (status) {
+    return (
+      <>
+        <td><span className={statusActive}>Active Member</span></td>
+        <td>
+          <button type="button" className={`${btnMission} ${btnLeave}`} onClick={handleStatus}>
+            Leave Mission
+          </button>
+        </td>
```
Comment (Have-Samuel) [inline src/components/missions/Status.jsx:18]: Nice use of the table element in your code. :100:

**Review by Have-Samuel — CHANGES_REQUESTED** [review]
Excellent work done so far :1st_place_medal: but you need to make a few changes to the code, check out my comments below for more clarity.

And feel free to DM me in case of any issues

File: src/App.js (hunk 1 above)
Comment (TesheMaximillan) [reply src/App.js:5]: 👋 Hi, Have Samuel, 

# 
Thank you for your suggestion, however, to get a workable Navbar we need to add every router component, and Myprofile is one of the router components, so, when we add anything in each route we can easily navigate to each router components. Also, this is why we put it together in the first place

**Review by Have-Samuel — APPROVED** [review]
Excellent work done :100: Approved :+1: 
Nothing else is needed, go ahead and merge

**Pull request merged**